nginx-pfe-staging-redirects.conf

```diff
@@ -0,0 +1,84 @@
+    include /etc/nginx/az-proxy-headers.conf;
+}
+
+# User profile page
+location ~* ^/users/[\w-]+/?$ {
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$proxy_path$request_uri;
+
+    resolver 1.1.1.1;
+    proxy_pass https://zooniversestatic.z13.web.core.windows.net/$proxy_path/;
+    include /etc/nginx/az-proxy-headers.conf;
+}
+
+# User specific pages
+location ~* ^/users/[\w-]+/((collections|favorites|message)?)/?$ {
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$proxy_path$request_uri;
+
+    resolver 1.1.1.1;
+    proxy_pass https://zooniversestatic.z13.web.core.windows.net/$proxy_path/;
+    include /etc/nginx/az-proxy-headers.conf;
+}
+
+# Default: /project/* to PFE
+# Can be overridden by directives in nginx-project-redirects.conf
+location ~* ^/projects {
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$proxy_path$request_uri;
+
+    resolver 1.1.1.1;
+    proxy_pass https://zooniversestatic.z13.web.core.windows.net/$proxy_path/;
+    include /etc/nginx/az-proxy-headers.conf;
+}
+
+# Most of the main PFE redirects
+location ~* ^/(organizations|collections|favorites|talk|notifications|inbox|lab|admin|accounts|reset-password|settings|privacy|security|youth_privacy) {
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$proxy_path$request_uri;
+
+    resolver 1.1.1.1;
+    proxy_pass https://zooniversestatic.z13.web.core.windows.net/$proxy_path/;
+    include /etc/nginx/az-proxy-headers.conf;
+}
+
+# unsubscribe route uses redirects between panoptes and the UI code
+# so needs it's own location block to handle the form submission POST
+# and the GET page loading (PFE routing handles the path)
+location /unsubscribe {
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$proxy_path$request_uri;
+
+    resolver 1.1.1.1;
+    if ($request_method ~ ^(GET|HEAD)$) {
+        proxy_pass             https://zooniversestatic.z13.web.core.windows.net/$proxy_path/;
+        set $proxy_host_header "zooniversestatic.z13.web.core.windows.net";
+    }
+    if ($request_method = POST) {
+        proxy_pass             https://panoptes.zooniverse.org$request_uri;
+        set $proxy_host_header "panoptes.zooniverse.org";
+    }
+    proxy_set_header Host $proxy_host_header;
+    proxy_redirect         /$host/ /;
+
+    include /etc/nginx/az-proxy-headers.conf;
+}
+
+# Example FEM project redirect
+location ~* ^/projects/(?:[\w-]*?/)?brooke/i-fancy-cats/?(?:(classify|about)(?:/.+?)?)?/?$ {
+    resolver 1.1.1.1;
+    proxy_pass "https://fe-project.preview.zooniverse.org";
```

Comment:
Just flagging one specific case to ask a syntax question: should direct assignment of `proxy_pass` URLs be written without double quotes (this one here uses quotes, as do a few other places in this PR). Acknowledging and guessing it probably doesn't matter, but wanted to check.

Reply:
As far as I know it doesn't matter and we're just kind of inconsistent in this repo. You need quotes to avoid escaping certain characters but otherwise it works either way.